packages/@okta/vuepress-theme-default/assets/css/okta/base/layout.scss

```diff
@@ -6,10 +6,6 @@ body {
     background: get-color('white');
     position: relative;
 
-  @include media('<medium') {
```

Comment:
Why was this removed?

Reply:
The top padding was pushing the search bar down below the header. With the top padding rule, the search bar was not hidden by default and then just moved down when the search icon was clicked resulting in this.

<img width="793" alt="Screen Shot 2020-01-29 at 1-05-14-613-PM" src="https://user-images.githubusercontent.com/742582/73388547-0edc4400-4298-11ea-99a7-ccd46b38f11b.png">